docs/examples/nlohmann_json_serialize_enum.cpp

```diff
@@ -26,7 +26,7 @@ enum class Color
     red, green, blue, unknown
 };
 
-NLOHMANN_JSON_SERIALIZE_ENUM(Color,
```

Comment:
Please add a new example instead of editing this one.

Reply:
Done.